### ISSUE TITLE
[tools] Validator Quality of Life tools

### ISSUE THREAD

**Issue by 0o-de-lally** [OLSF/libra#1123]
Validator Quality of Life

1. - [ ] Hourly snapshots to be saved to epoch-archive [@0o-de-lally 
2. - [ ] `ol restore` to allow setting a specific version/height number, otherwise always fetch the highest version of the highest epoch. [@0o-de-lally
3. - [ ] A single validator.node.yaml file, which can serve the validator for when it is in the set, or out of the set. [No Owner]
4. - [ ] A recommended systemd daemon file, which restarts the node every hour, and restarts if the node fails. [No Owner]

**Comment by sirouk** [comment]
I would like to propose a single validator.node.yaml file that can serve validator syncing for all cases. Please let me know how we can proceed.

**Comment by hemulin** [comment]
@sirouk 

Has anything happened with that proposed yaml file?

**Comment by hemulin** [comment]
@0o-de-lally 

Isn't [this file](https://github.com/OLSF/libra/blob/main/ol/documentation/node-ops/validators/validator_daemon_configs.md) is speaking of the open request for the systemd process?

**Comment by sirouk** [comment]
@hemulin yes, the unified yaml is validator.node.yaml and can be generated anew via ol init --val
